modules/nw-metalLB-basic-upgrade-operator.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-upgrading-operator.adoc
+
+:_content-type: PROCEDURE
+
+[id="upgrading-metallb-operator_{context}"]
+= Upgrading the MetalLB Operator
+
+You can manually upgrade the MetalLB Operator from a version less than {product-version} by using the {product-title} web console. There are a couple of ways to delete Operators from a cluster and these are described in the additional resources section. 
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Access the web console and navigate to *Operators* -> *Installed Operators*.
+
+. Click *MetalLB Operator* to open the *Operator details* page.
+
+. Update the *Update approval* strategy.
++
+[NOTE]
+====
+When updating from version less than 4.11 set the *Update approval* strategy to manual. The automatic update mechanism is not working as expected.
+====
+
+.. Click *Subscription* to reveal the *Subscription details*. 
+
+.. Click the pencil icon associated with the *Update approval* option. Select *Manual*, and click *Save*. 
+
+. In the same *Subscription* screen click the *Actions* drop down and click *Uninstall Operator*.
```

Comment:
Here we are documenting how to remove via the console.  I'd either copy a whole section about removing the operator here OR just defer to the official docs and say "remove the operator".